blog/content/docs/basics.adoc

```diff
@@ -331,6 +331,47 @@ Ok, to dive a bit deeper, we could create a json listing all posts with some inf
 
 {|
 
+=== How to create custom collections?
+
+You can easily create your own collection, such as documentation, recipes, team members, or conference talks. To do this, simply create a new folder under the `content` directory. For example, if you're adding `docs`, it would look like this:
+
+[source]
+----
+content/
+├── docs
+│   ├── 01-chap
+│   │   ├── image1.png
+│   │   └── index.adoc
+│   ├── 02-chap
+│   │   ├── image2.png
+│   │   ├── index.adoc
+└── posts
+    └── 2025-01-02-my-first-blog
+        └── index.md
+----
+
+In this example, we have two collections: `posts` and `docs`. Finally, you need to define the new collection in the `config/application.properties` file using the following properties:
+
+[source]
+----
+site.collections.docs=true
+site.collections.docs.layout=":theme/post"
```

Comment:
You probably needs the `site.collections.docs.future=true` as this in not a time-based collection (I need to work on this at some point to allow specifying different sorting options)..